Bottom: eb35bccb6ed76896fb86cd18bdc7ef9d7c235193
Top:    7d6dcb5a174620866d1d226cd0a6aec854fb6e73
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:27:06 +0100

Refresh of draw-c-expose-some-drawing-api

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 4d9fdbe..1c5c150 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1928,7 +1928,8 @@ N_("Bring up the popup menu specified by @code{MenuName}.\n"
 
 This just pops up the specified menu.  The menu must have been defined
 as a named subresource of the Popups resource in the menu resource
-file.
+file. The second, optional (and ignored) argument represents the mouse
+button number which is triggering the popup.
 
 %end-doc */
